docs/source/specs/openapi.yml

```diff
@@ -892,6 +1063,87 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/ProviderOut'
+    ReportGroupingCost:
+      type: string
+      enum:
+        - account
+        - service
+        - instance_type
+        - storage_type
+    ReportGroupingInstance:
+      type: string
+      enum:
+        - account
+        - instance_type
+    ReportGroupingStorage:
+      type: string
+      enum:
+        - account
+        - storage_type
+    ReportResolution:
+      type: string
+      enum:
+        - daily
+        - monthly
+    ReportTimeScope:
```

Comment:
Maybe it's just me but I'm not sure how I feel about the negative numbers.  Is there any interesting background on this?

Reply:
I used a limited set from the Cloud Health API as the inspiration.
http://apidocs.cloudhealthtech.com/#reporting_understand-time-filters